pkg/process/procutil/process_linux.go

```diff
@@ -611,10 +611,13 @@ func (p *probe) parseStatContent(statContent []byte, sInfo *statInfo, pid int32,
 		sInfo.ppid = int32(ppid)
 	}
 
-	flags, err := strconv.ParseInt(flagStr, 10, 32)
+	flags, err := strconv.ParseUint(flagStr, 10, 32)
 	if err == nil {
 		sInfo.flags = uint32(flags)
 	}
+	if sInfo.flags == 0 {
+		log.Warnf("unable to parse flags for pid %d", pid)
+	}
```

Comment:
Is 0 not a possible value?

Reply:
Added the warn log just in case. But, `0` is not a possible value, then removed it.